src/Generator/IdGenerator.php

```diff
@@ -35,11 +35,15 @@ public function __construct(int $firstLevel = 100000, int $secondLevel = 1000)
 
     public function generatePath(MediaInterface $media): string
     {
-        $mediaId = (int) $media->getId();
+        $id = $media->getId();
 
-        $rep_first_level = (int) ($mediaId / $this->firstLevel);
-        $rep_second_level = (int) (($mediaId - ($rep_first_level * $this->firstLevel)) / $this->secondLevel);
+        if (!is_numeric($id)) {
+            return '';
```

Comment:
Does it make sense to generate a Exception here? Maybe yes, because it wouldn't make sense to generate a path without the id of the media right?

Reply:
It make sens to me to generate something like an InvalidArgument.